docs/concepts/cli.mdx

```diff
@@ -146,3 +146,32 @@ crewai run
 Make sure to run these commands from the directory where your CrewAI project is set up. 
 Some commands may require additional configuration or setup within your project structure.
 </Note>
+
+
+### 9. API Keys
+
+When running ```crewai create crew``` command, the CLI will prompt for API keys. It will first show you the top 5 most common LLM providers and ask you to select one.
```

Comment:
When running ```crewai create crew``` command, the CLI will first show you the top 5 most common LLM providers and ask you to select one.

Once you've selected an LLM provider,  you will be prompted for API keys.